app/views/contexts/calculated_summary_context.py

```diff
@@ -253,9 +288,7 @@ def _get_answer_format(self, groups: list) -> Tuple[dict[str, Any], list]:
         return answer_format, values_to_calculate
 
     @staticmethod
-    def _format_total(
-        answer_format: Mapping[str, str], total: Union[int, float, Decimal]
-    ) -> str:
+    def _format_total(answer_format: Mapping[str, str], total: NumericType) -> str:
```

Comment:
Since we are refactoring these we should probably force positional arguments. Same below.

Reply:
good shout 👍